SimpleWifi/Wifi.cs

```diff
@@ -29,7 +29,13 @@ public Wifi()
 			foreach (var inte in _client.Interfaces)
 				inte.WlanNotification += inte_WlanNotification;
 		}
-		
+		public void OnQuit()
+		{
+			foreach (var inte in _client.Interfaces)
+				inte.WlanNotification -= inte_WlanNotification;
```

Comment:
Could you add { } here like you have don in the next foreach and give a better name to "inte". For example:

`foreach (var interface in _client.Interfaces)
{
    interface.WlanNotification -= inte_WlanNotification;
}
`

Reply:
Changes added.